src/flaskr/templates/base.html

```diff
@@ -9,6 +9,11 @@
 </head>
 <body>
     <header class="top-header">
+        <style>
+            body {
+                font-family: Arial, sans-serif;
+            }
+        </style>
         <h1>{% block header %}Manufacturing Line Capacity Prediction{% endblock %}</h1>
 
         <h2>{% block header2 %}{% if current_user.is_anonymous %}{% else %}Hello {{current_user.username}}{% endif %}{% endblock %}</h2>
```

Comment:
Should we center this? Right now it appears a little off-center.

Reply:
Which part is off-centre? I was trying to fix the Login Page Title to match the rest but it wasn't working for me. Could you send me what you're seeing?

Reply:
![image](https://github.com/Birdbh/Engicoders/assets/88523672/8096064d-1063-4897-9a05-8bbb7d92d0e4)

Reply:
Off-centre on an ultrawide display. The given html would likely fix it, mentioning it here as you're already making the changes to the affected file here.